tests/testthat/helper-HTML.R

```diff
@@ -76,19 +76,23 @@ get_grid_lines <- function(html, p_name, grid_class){
   attr_v <- apply(attr_v, 2, as.numeric)
   return(list(hor=attr_h, vert=attr_v))
 }
+### The hex codes come from
+### https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values
+key2hex_code <- c(
```

Comment:
hi @siddhesh195 here is my attempt to clarify the documentation for the key codes, what do you think?

Reply:
yes. it looks good